<launch>
	<node name="eddie_ssl_trainer" pkg="eddiebot_ssl" type="trainer.py" output="screen">
		<param name="trace_file" value="$(find eddiebot_ssl)/assets/3room_trace.json" />
		<param name="room_dest" value="$(find eddiebot_ssl)/assets/room_dest.json" />
		<param name="person_loc" value="$(find eddiebot_ssl)/assets/person_loc.json" />
	</node>
</launch>